.gitignore

```diff
@@ -1,16 +0,0 @@
-.snakemake/*
```

Comment:
@axiomcura Thanks, and apologies, I more meant the file in entirety rather than just this line. Stated another way: would it make sense to keep all or some of this `.gitignore` file?

Reply:
Oh! I see what you mean. No, there's no reason to keep the `.gitignore` file. Thanks 😂

Reply:
For some odd reason, I cannot remove the `.gitignore` file directly from the PR. I'll remove it in the upcoming PR.

Reply:
we like `.gitignore`! why remove :(

gitignore keeps our git environments clean, which makes realizing our changes easier

Reply:
That is a good point. For now, I'll keep it.